cmd/activator/main.go

```diff
@@ -174,11 +175,17 @@ func main() {
 	endpointInformer := kubeInformerFactory.Core().V1().Endpoints()
 	serviceInformer := kubeInformerFactory.Core().V1().Services()
 	revisionInformer := servingInformerFactory.Serving().V1alpha1().Revisions()
+	sksInformer := servingInformerFactory.Networking().V1alpha1().ServerlessServices()
 
 	// Run informers instead of starting them from the factory to prevent the sync hanging because of empty handler.
-	go revisionInformer.Informer().Run(stopCh)
-	go endpointInformer.Informer().Run(stopCh)
-	go serviceInformer.Informer().Run(stopCh)
+	if err := controller.StartInformers(
+		stopCh,
+		revisionInformer.Informer(),
+		endpointInformer.Informer(),
+		serviceInformer.Informer(),
+		sksInformer.Informer()); err != nil {
+		logger.Fatalf("Failed to start informers: %v", err)
+	}
```

Comment:
You can now remove the `informerSyncs` code below. `StartInformers` will already wait for a cache sync.

Reply:
hah, I was sure I removed it :)